### PR TITLE
Formatting, words

### PR DESCRIPTION
**Pull request by robatron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,16 @@ A developer whom would like to propose a change to a code style guide must:
...
+    - If needed, modify your PR per team recommendations and notify team of updates.
+1. Email the team requesting a vote, open for 1 week.
 1. Merge the PR after:
     - A minimum of 2/3 of team indicate approval<sup>1</sup>, at least one of whom must be a senior<sup>2</sup>, and
-    - At least 1 week's time has passed from initial presentation to give everyone a chance to review
+    - At least 1 week's time has passed from request-vote email<sup>3</sup>.
 1. Email the team with the update.
 
-<sup>1</sup>No-votes will not be included in the calculation.<br>
+<sup>1</sup>Obstains will not be included from the approval calculation.<br>
```
Comment (petersgrandadventure) [inline README.md]: Abstain

Comment (peripateticus) [inline README.md]: from -> in?

Comment (robatron) [inline README.md]: Fixed, and fixed

**Comment by agats** [comment]
+1
Thanks Rob!

**Pull request merged**